src/storage/page_buffer.h

```diff
@@ -166,6 +166,8 @@ extern const VPID vpid_Null_vpid;
 #define PGBUF_IS_ORDERED_PAGETYPE(ptype) \
   ((ptype) == PAGE_HEAP || (ptype) == PAGE_OVERFLOW)
 
+// check if page has change based on current LSA and a previous reference LSA 
+#define PGBUF_PAGE_HAS_CHANGED(pgptr, ref_lsa)    (!LSA_EQ ((ref_lsa), pgbuf_get_lsa ((pgptr))))
```

Comment:
NIT:

I think PGBUF_IS_PAGE_CHANGED is more readable.

Reply:
OK, Thank you.